### ISSUE TITLE
Password rest

### ISSUE THREAD

**Issue by sathyajackson** [kevinpapst/kimai2#1566]
Expected behaviour
Valid Password reset functionality

Actual behaviour
getting 500 Error with "Something Went Wrong"

Steps to reproduce
Home screen --> Password reset --> hit the change password link from mail ---> Update the password and boom the error pops up.

Reproducible on demo.kimai.org? [yes|no]
no

Kimai version [for example: dev 1.1.0.1389|github master|github develop]
Relevant log entries from log file (debug tab or temporary/logfile.txt)
[2020-03-17 10:04:06] request.INFO: Matched route "fos_user_resetting_reset". {"route":"fos_user_resetting_reset","route_parameters":{"_route":"fos_user_resetting_reset","_controller":"fos_user.resetting.controller:resetAction","_locale":"en","token":"5IH2FRMIT8tjS68dO-WW380xDQFxKhzbJqgEuEz-0Wc"},"request_uri":"http://timesheet.xxxx.com/en/resetting/reset/5IH2FRMIT8tjS68dO-WW380xDQFxKhzbJqgEuEz-0Wc","method":"POST"} []
[2020-03-17 10:04:06] security.DEBUG: Checking for guard authentication credentials. {"firewall_key":"secured_area","authenticators":1} []
[2020-03-17 10:04:06] security.DEBUG: Checking support on guard authenticator. {"firewall_key":"secured_area","authenticator":"App\Security\TokenAuthenticator"} []
[2020-03-17 10:04:06] security.DEBUG: Guard authenticator does not support the request. {"firewall_key":"secured_area","authenticator":"App\Security\TokenAuthenticator"} []
[2020-03-17 10:04:06] security.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2020-03-17 10:04:06] php.INFO: User Deprecated: Calling the "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" method with the event name as the first argument is deprecated since Symfony 4.3, pass it as the second argument and provide the event object as the first argument instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Calling the "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" method with the event name as the first argument is deprecated since Symfony 4.3, pass it as the second argument and provide the event object as the first argument instead. at /var/www/kimai2/vendor/symfony/event-dispatcher/EventDispatcher.php:58)"} []
[2020-03-17 10:04:06] php.INFO: User Deprecated: Calling the "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" method with the event name as the first argument is deprecated since Symfony 4.3, pass it as the second argument and provide the event object as the first argument instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Calling the "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" method with the event name as the first argument is deprecated since Symfony 4.3, pass it as the second argument and provide the event object as the first argument instead. at /var/www/kimai2/vendor/symfony/event-dispatcher/EventDispatcher.php:58)"} []
[2020-03-17 10:04:07] php.INFO: User Deprecated: Calling the "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" method with the event name as the first argument is deprecated since Symfony 4.3, pass it as the second argument and provide the event object as the first argument instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Calling the "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" method with the event name as the first argument is deprecated since Symfony 4.3, pass it as the second argument and provide the event object as the first argument instead. at /var/www/kimai2/vendor/symfony/event-dispatcher/EventDispatcher.php:58)"} []
[2020-03-17 10:04:07] security.DEBUG: Clearing remember-me cookie. {"name":"REMEMBERME"} []
[2020-03-17 10:04:07] security.DEBUG: Did not send remember-me cookie. {"parameter":"_remember_me"} []
[2020-03-17 10:04:07] security.DEBUG: Remember-me was not requested. [] []
[2020-03-17 10:04:07] php.INFO: User Deprecated: Calling the "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" method with the event name as the first argument is deprecated since Symfony 4.3, pass it as the second argument and provide the event object as the first argument instead. {"exception":"[object] (ErrorException(code: 0): User Deprecated: Calling the "Symfony\Component\EventDispatcher\EventDispatcherInterface::dispatch()" method with the event name as the first argument is deprecated since Symfony 4.3, pass it as the second argument and provide the event object as the first argument instead. at /var/www/kimai2/vendor/symfony/event-dispatcher/EventDispatcher.php:58)"} []
[2020-03-17 10:04:07] php.CRITICAL: Uncaught Error: Argument 2 passed to FOS\UserBundle\EventListener\FlashListener::__construct() must implement interface Symfony\Component\Translation\TranslatorInterface, instance of App\Utils\Translator given, called in /var/www/kimai2/var/cache/prod/ContainerSMcvoaE/getFosUser_Listener_FlashService.php on line 11 {"exception":"[object] (TypeError(code: 0): Argument 2 passed to FOS\UserBundle\EventListener\FlashListener::__construct() must implement interface Symfony\Component\Translation\TranslatorInterface, instance of App\Utils\Translator given, called in /var/www/kimai2/var/cache/prod/ContainerSMcvoaE/getFosUser_Listener_FlashService.php on line 11 at /var/www/kimai2/vendor/friendsofsymfony/user-bundle/EventListener/FlashListener.php:48)"} []
[2020-03-17 10:04:07] request.CRITICAL: Uncaught PHP Exception TypeError: "Argument 2 passed to FOS\UserBundle\EventListener\FlashListener::__construct() must implement interface Symfony\Component\Translation\TranslatorInterface, instance of App\Utils\Translator given, called in /var/www/kimai2/var/cache/prod/ContainerSMcvoaE/getFosUser_Listener_FlashService.php on line 11" at /var/www/kimai2/vendor/friendsofsymfony/user-bundle/EventListener/FlashListener.php line 48 {"exception":"[object] (TypeError(code: 0): Argument 2 passed to FOS\UserBundle\EventListener\FlashListener::__construct() must implement interface Symfony\Component\Translation\TranslatorInterface, instance of App\Utils\Translator given, called in /var/www/kimai2/var/cache/prod/ContainerSMcvoaE/getFosUser_Listener_FlashService.php on line 11 at /var/www/kimai2/vendor/friendsofsymfony/user-bundle/EventListener/FlashListener.php:48)"} []
[2020-03-17 10:04:07] security.DEBUG: Stored the security token in the session. {"key":"_security_secured_area"} []

**Comment by kevinpapst** [comment]
Seems to be a bug. Will be fixed with the next version.

**Comment by sathyajackson** [comment]
Thanks Kevin, do we have any temporary fix as to not show the 500 internal server error message and redirect the user to login screen or something like that ?

The tool is deployed on prod and ensures are raising this error as a big concern.

**Comment by kevinpapst** [comment]
You would have to patch a file in vendor/friendsofsymfony/user-bundle/EventListener/FlashListener.php and replace in line 18(?)
```
use Symfony\Component\Translation\TranslatorInterface;
```
with 
```
use Symfony\Contracts\Translation\TranslatorInterface;
```
but I cannot promise that it works. I have to have a closer look.

**Comment by kevinpapst** [comment]
Fixed with #1568 1568
